resources/graphClient/graphClient.html

```diff
@@ -10,19 +10,21 @@
     This is the HTML for exploring graphs, and is hosted inside a WebView in a VS HTML preview window
 -->
 
+<!--The xyzUri strings in the scripts' and icon's src field will be replaced by the approrpriate strings at runtime when this html file is read -->
+
 <head>
-    <link rel="stylesheet" type="text/css" href="graphClient.css">
+    <link rel="stylesheet" type="text/css" href="cssUri">
```

Comment:
I'd prefer something that stands out more as being something to get replaced, like $CSSURI$.

It also seems like it would be easier to just have a single $BASEURL$ replacement and use src="$BASEURL/dist/node_modules/d3/d3.js" etc.

Reply:
I didn't take that route since it looked like the paths would work differently on Windows : 
![image](https://user-images.githubusercontent.com/30420973/54565718-3c741380-498c-11e9-9fcd-17d809e67ce2.png)


But it looks like it doesn't matter whether '\\' or '/' is used. Your approach looks easier to maintain.

Reply:
Looks like it works.